cbor.js

```diff
@@ -172,21 +311,24 @@ function encode(value) {
   }
 
   encodeItem(value);
-
+  /* istanbul ignore if */
   if ("slice" in data)
```

Comment:
this is a feature check that can't get coverage testing; it needs to be excluded or needs to be removed